package.json

```diff
@@ -17,7 +17,9 @@
     "check": "npm run lint && npm test && npm run audit && npm outdated --depth 0",
     "prebuild": "rimraf build",
     "build": "babel lib --out-dir build",
-    "prepublish": "npm run build"
+    "prepublish": "npm run build",
+    "precommit": "npm run lint",
+    "prepush": "npm run validate"
```

Comment:
I don't have any problems with these pre-defined hooks. They are quite explicit. I love them much more then a fuzzy-matching way of the `pre-commit` lib.